hera/editor/lib/updateUndoStack.tsx

```diff
@@ -1,5 +1,11 @@
 import { EditorState, SetEditorStateFunction, UndoEntry } from '../Types.tsx';
 
+export const getUndoStackKey = (id: string | undefined) =>
+  `map-editor-undo-stack-${id ? `${id}` : 'fallback'}`;
+
+export const getUndoStackIndexKey = (id: string | undefined) =>
+  `map-editor-undo-stack-index-${id ? `${id}` : 'fallback'}`;
+
```

Comment:
Just a suggestion, curious what you think: From reading the code it's not obvious why the `id` is passed. What do you think about calling these `getUndoStackKeyFor(id)` and `getUndoStackIndexKeyFor(id)`?

Sorry, didn't come to me earlier since they were uppercased and that confused me 😅

Reply:
Hah, not a problem at all. Definitely think adding the `For` makes the intent a little clearer